src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: max-content;
+  font-weight: 400;
+  padding: 32px $card-bottom-right-padding $card-bottom-right-padding;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__photo {
+    align-self: center;
```

Comment:
add `display: block;` and horizontal `margin: auto;`, instead of `align-self: center;`

Reply:
The same situation as previous comment